containers/ecr-viewer/src/app/components/EcrFilters.tsx

```diff
@@ -216,6 +167,45 @@ const FilterReportableConditions = ({
  * - Updates the browser's query string when the filter is applied.
  */
 const FilterByDate = ({ initCustomDate, initDateRange }: FilterProps) => {
+  const CustomDateInput = ({
```

Comment:
the thinking on why this was currently in the `BaseFilter` file as it's designed to be a re-usable component. Putting it inside the `FilterByDate` really loses this. I'm okay with co-locating it file-wise given no one else is yet using it, but would really rather it not be encapsulated in another function

Reply:
looks like this was already reverted